db/migrate/20160721225722_add_owner_to_embeddable_media.rb

```diff
@@ -0,0 +1,6 @@
+class AddOwnerToEmbeddableMedia < ActiveRecord::Migration
+  def change
+    add_reference :embeddable_media, :owner, polymorphic: true, index: true
+    add_column :embeddable_media, :owner_attribute, :string
```

Comment:
What are the different types of owners and attributes here. Do we really need to add more complexity to the data model?